sechub-sereco/src/test/java/com/mercedesbenz/sechub/sereco/WorkspaceTest.java

```diff
@@ -59,7 +60,7 @@ void do_import__no_importer() throws IOException {
         /* formatter:on */
```

Comment:
you have added the masking service as a new dependency.
But we have no test that uses a mocked masking service...

Please add a masking service mock inside the test and
- check the masking service is called by workspace test when it comes to merging
- check merging uses the masked parts and not the origin ones (I would just check that the list from mocked service is used...)